streamer/src/recvmmsg.rs

```diff
@@ -75,8 +75,8 @@ pub fn recv_mmsg(sock: &UdpSocket, packets: &mut [Packet]) -> io::Result<(usize,
                     let mut p = &mut packets[i];
                     p.meta.size = hdrs[i].msg_len as usize;
                     total_size += p.meta.size;
-                    let inet_addr = InetAddr::V4(addr[i]);
-                    p.meta.set_addr(&inet_addr.to_std());
+                    //let inet_addr = InetAddr::V4(addr[i]);
+                    //p.meta.set_addr(&inet_addr.to_std());
```

Comment:
note that I'm not sure these are safe to omit to set_addr....

Reply:
iirc, my thinking were like this: udp src are spoof-able, so we shouldn't trust it to begin with as a blockchain node... lol

Reply:
Is the address for incoming packets used? I didn't see any consumers.

Reply:
it seems to be used?

the set_addr's counterpart (= `fn addr`) are called from various places.

![image](https://user-images.githubusercontent.com/117807/125747561-f3018263-f94e-4ead-8746-5006119db3f5.png)